yajsapi/executor/index.ts

```diff
@@ -589,28 +581,28 @@ export class Executor {
       consumer: Consumer<Task<D, R>>
     ): Promise<void> {
       /* TODO ctrl+c handling */
-      let { done = false, value: item }: { done?: boolean; value: WorkItem }
-        = await command_generator.next();
+      let { done = false, value: item }: { done?: boolean; value: WorkItem } = await command_generator.next();
       while (!done) {
         const [batch, exec_options] = unpack_work_item(item);
         if (batch.timeout()) {
           if (exec_options.batch_timeout) {
-            logger.warn(
-              "Overriding batch timeout set with commit({timeout:...}) by value from ExecOptions"
-            );
+            logger.warn("Overriding batch timeout set with commit({timeout:...}) by value from ExecOptions");
           } else {
             exec_options.batch_timeout = batch.timeout();
           }
         }
-        const batch_deadline
-          = exec_options.batch_timeout ? ((Date.now() + exec_options.batch_timeout) / 1000) : undefined;
+        const batch_deadline = exec_options.batch_timeout
```

Comment:
again, if we're splitting the ternary into multiple lines, shouldn't we replace it with a more conventional if/else?